### ISSUE TITLE
ERROR: shada file is not readable

### ISSUE THREAD

**Issue by jrock2004** [neovim/neovim#13218]
- `nvim --version`: 0.4.4
- `vim -u DEFAULTS` (version: ) behaves differently?
- Operating system/version: Mac 10.15.7
- Terminal name/version: iTerm2
- `$TERM`: screen-256color

### Steps to reproduce using `nvim -u NORC`

```
nvim -u NORC
# Alternative for shell-related problems:
# env -i TERM=ansi-256color "$(which nvim)"

:chechhealth
```

I saw issue #12407 and someone mention system issues. In ` /Users/jcostanzo/.local/share/nvim/` directory there is not a shada dir. 

### Actual behaviour

![image](https://user-images.githubusercontent.com/655716/98004794-5612da80-1dbe-11eb-837c-c195a0ced118.png)

### Expected behaviour

Not see error

**Comment by subnut** [comment]
# Please correctly fill out the form given above

- Did you try `vim -u DEFAULTS`?
- `vim -u DEFAULTS` (version: **???** ) behaves differently?
- ### Steps to reproduce using `nvim -u NORC`?

**Comment by jrock2004** [comment]
So it seems with `nvim -u DEFAULTS` I do not get the error. Guess I need to hunt, what in my file could be causing this

**Comment by subnut** [comment]
@jrock2004 See, this is exactly why that initial form is given. It helps us avoid unnecessary issues.
I advise you to close this issue. In case you find something that needs our attention, you can reopen it.
<br>
Regards
subnut

**Comment by erw7** [comment]
@jrock2004 Do you have either of the following settings in place?

- `shadafile` is set to `NONE` or `-i NONE` is set as a command line option at startup.
- Set shada file with the `shada (viminfo)` option `n`.

In that case, `:checkhelath` could show a false error. Fixed at #13291. Does #13291 solve your problem?

**Comment by Universal-Invariant** [comment]
I'm getting the same error with the latest neovim. It seems that it's looking in the wrong directory. In windows I'm seeing the shada dir in nvim\Kickstart\Kickstart-data. Similarly for Lazy and AstroVim: .state\nvim\AstroNvim/shada/main.shada but it's in .state\nvim\AstroNvim\AstroNvim-data/shada/main.shada.

I thought about linking to it but not sure if it is just an false error due to paths in the health check or if it is a real error since the data actually exists. Not 100% sure if it's being used since I'm new to neovim and still setting things up. I also created a custom lua router which modifies the paths to try to put everything in subdirectories under nvim so it could something related to that. The issue here suggests it may be due to the path being hard coded in the health check and unrelated to my router code.
